src/styles/blocks/page.scss

```diff
@@ -0,0 +1,9 @@
+.page {
+  &__body {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: $page-height;
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), variables should be used for colors and font-sizes, but not for setting heights and widths directly. Consider using a more descriptive variable or a calculation based on the root font size if the height needs to be responsive.